u-root.go

```diff
@@ -136,11 +147,74 @@ func generateLabel() string {
 	return fmt.Sprintf("%s-%s-%s-%s", *build, env.GOOS, env.GOARCH, strings.Join(baseCmds, "_"))
 }
 
+func gobusyboxMain() error {
+	bopts := &golang.BuildOpts{}
+	// hmm.
+	//	bopts.RegisterFlags(flag.CommandLine)
```

Comment:
I think you need to add the *tags somewhere in here.

Reply:
hmm. where :-)